scripts/cron.py

```diff
@@ -50,76 +53,89 @@ def log(self, message):
 
 
 # Every 10 minutes.
-@scheduled_job('cron', month='*', day='*', hour='*', minute='*/10', max_instances=1, coalesce=True)
+@scheduled_job('cron', month='*', day='*', hour='*', minute='*/10',
+               max_instances=1, coalesce=True)
 @babis.decorator(ping_after=settings.DMS_ENQUEUE_LAG_MONITOR_TASK)
 def job_enqueue_lag_monitor_task():
     call_command('cron enqueue_lag_monitor_task')
 
 
 # Every hour.
-@scheduled_job('cron', month='*', day='*', hour='*', minute='30', max_instances=1, coalesce=True)
+@scheduled_job('cron', month='*', day='*', hour='*', minute='30',
+               max_instances=1, coalesce=True, skip=settings.READ_ONLY)
 @babis.decorator(ping_after=settings.DMS_SEND_WELCOME_EMAILS)
 def job_send_welcome_emails():
     call_command('cron send_welcome_emails')
 
 
-@scheduled_job('cron', month='*', day='*', hour='*', minute='59', max_instances=1, coalesce=True)
+@scheduled_job('cron', month='*', day='*', hour='*', minute='59',
+               max_instances=1, coalesce=True, skip=(settings.READ_ONLY | settings.STAGE))
```

Comment:
shouldn't this be `(settings.READ_ONLY or settings.STAGE)`? I'm sure this works, but it seems a bit less readable than just the or operator. Just a NIT though, not a blocker if you really prefer it this way.

Reply:
Sure, why not. updated